DEVELOPMENT.md

```diff
@@ -58,42 +58,29 @@ We use a mix of AWS and GCP to host these clusters for diversity. Typically you
 
 * Create a new testing branch on your fork of the `opendatahub-io/ai-edge` repo, preferably named `<your-kerberos-name>-acm-dev`.
 * Ensure that you change `namespace` in either `test/gitops/bike-rental-app/kustomization.yaml` or `test/gitops/tensorflow-housing-app/kustomization.yaml`, or both, as appropriate.
-* Substitute the values of the `GIT_REPO_URL`, `GIT_BRANCH`, and `TEST_NAMESPACE` variables in this `make` command appropriately:
+* Substitute the values of the `GIT_REPO_URL`, `GIT_BRANCH`, `CUSTOM_PREFIX`, and `CUSTOM_APP_NAMESPACE` variables in this `make` command appropriately:
```

Comment:
I changed `TEST_NAMESPACE` to `CUSTOM_APP_NAMESPACE` here because previously we used a shared namespace name for resources on both the hub and managed clusters, whereas with this change, the resources on the hub are applied to the `openshift-gitops` namespace.

Given that everything on the hub goes to the `openshift-gitops`, `CUSTOM_PREFIX` is needed to prevent clobbering resources from other instances of the app that you may have deployed. All of this might be less necessary now than it was when we used a shared dev cluster, but I can imagine a scenario where you've got an instance deployed when you're working on something, but then you may want to deploy an instance to test out changes in a PR from a colleague.